mosec/log.py

```diff
@@ -22,9 +22,10 @@
 from datetime import datetime
 from typing import Any, MutableMapping
 
-from mosec.args import mosec_args
+from .args import mosec_args
 
 MOSEC_LOG_NAME = __name__
+USER_LOG_NAME = "MOSEC_USER_LOG"
```

Comment:
If it's used by `logging.getLogger()`, then it should follow the same spec. (although this path may not exist)